editor/src/core/shared/template-path.ts

```diff
@@ -262,18 +272,25 @@ export function filterScenes(paths: TemplatePath[]): StaticInstancePath | Instan
 // FIXME: This should be retired, it's just plain dangerous.
 // Right now this is only used for SpecialNodes (in CanvasMetaData)
 function fromStringUncached(path: string): TemplatePath {
-  const [sceneStr, elementStr] = path.split(SceneSeparator)
-  const scene = sceneStr.split(ElementSeparator).filter((e) => e.length > 0)
+  let elementPathStrs = path.split(SceneSeparator)
+  const lastElementStr = elementPathStrs.pop()! // We know this is safe because ''.split(SceneSeparator).pop() returns ''
```

Comment:
I _think_ latest TypeScript has some stuff that allows you to do a spread like this: `const [...elementPathStrs, lastElementStr] = ...`.

Reply:
I just checked this and it appears to not be a thing (though I know you can do `const [head, ...tail] = [1,2,3,4]`)